build/run-e2e-tests.sh

```diff
@@ -2,5 +2,154 @@
+    	curl -Lo ./kind "https://kind.sigs.k8s.io/dl/v0.7.0/kind-$(uname)-amd64"
+    	chmod +x ./kind
+    	sudo cp ./kind /usr/local/bin/kind
+    fi
+    echo "Delete hub if it exists"
+    kind delete cluster --name hub || true
+    
+    echo "Start hub cluster" 
+    rm -rf $HOME/.kube/kind-config-hub
+    kind create cluster --kubeconfig $HOME/.kube/kind-config-hub --name hub --config ${WORKDIR}/test-data/e2e/kind-hub-config.yaml
+    # kubectl cluster-info --context kind-hub --kubeconfig $(pwd)/.kube/kind-config-hub # confirm connection 
+    export KUBECONFIG=$HOME/.kube/kind-config-hub
+	echo "KUBECONFIG" && echo $KUBECONFIG
+} 
+
+
+delete_kind_hub() {
+	echo "====Delete kind cluster====="
+    kind delete cluster --name hub
+}
+
+delete_command_binaries(){
+	cd ${WORKDIR}/..
+	echo "Current directory"
+	echo $(pwd)
+	rm ./kind
+	rm ./kubectl
+	rm ./oc
+}
+
+initial_setup() {
+    echo $WORKDIR
+    echo "=====Deploying insights-client====="
+	$sed_command "s~{{ INSIGHTS_CLIENT_IMAGE }}~$IMAGE_NAME~g" ./test-data/e2e/insights-chart/templates/insights-deployment.yaml
+	$sed_command "s~{{ INSIGHTS_CLIENT_CCX_TOKEN }}~$INSIGHTS_CLIENT_CCX_TOKEN~g" ./test-data/e2e/insights-chart/templates/insights-deployment.yaml
+    
+	echo "Current directory"
+	echo $(pwd)
+	echo -n "Create namespace open-cluster-management-monitoring: " && kubectl create namespace open-cluster-management
+	echo -n "Switch to namespace: " && kubectl config set-context --current --namespace open-cluster-management
+    echo -n "Creating pull secret: " && kubectl create secret docker-registry search-operator-pull-secret --docker-server=quay.io --docker-username=$DOCKER_USER --docker-password=$DOCKER_PASS
+
+    #create dummy ssl certs for service 
+	./setup.sh
+    kubectl create secret generic insights-client-certs --from-file=./sslcert/tls.crt --from-file=./sslcert/tls.key
+    echo -n "Applying cluster versions CRD:" && kubectl apply -f ./test-data/e2e/clusterversions.config.openshift.io.yaml
+    echo -n "Applying version CR:" && kubectl apply -f ./test-data/e2e/version.yaml
+    echo -n "Applying managedclusters CRD:" && kubectl apply -f ./test-data/e2e/managedclusters.yaml
+    echo -n "Applying local-cluster managedclusters CR:" && kubectl apply -f ./test-data/e2e/local-clusterCR.yaml
+
+    echo -n "Installing Insights client deployment :" && kubectl apply -f ./test-data/e2e/insights-chart/templates
+	cat ./test-data/e2e/insights-chart/templates/insights-deployment.yaml
+    sleep 100s
+	
+
+}
+
+test_content_and_local_cluster_report(){
+	pod=$(oc get pods | grep insights-client | cut -d' ' -f1)
+	oc logs $pod
+	log=$(oc logs $pod) 
+	log_msg=$(echo $log| grep "Creating Policy Info for cluster local-cluster (69365d80-c4ef-4999-8417-f1")
+	if [[ "$log_msg" == *"local-cluster"* ]]; then
+	    echo "Pod log has policy report"
+	else
+		echo "Pod log did not have expected output"
+	    exit 1
+    fi
```

Comment:
Should add a check to see if a PolicyReport CR was created... but not sure how we could make that happen